DOCS-12907 revision to VM/BM endpoint instructions (#568)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/deploy-to-runtime-fabric.adoc

. To preview the endpoint, click the generated preview link.
. To add additional endpoints, click *+ Add Endpoint*.

=== Configure a Public Endpoint for the Application for VMs/Bare Metal

Use the following steps to configure a public endpoint and network configuration for an application deployed to Runtime Fabric for VMs/Bare Metal.

You must first xref:enable-inbound-traffic.adoc[enable inbound traffic manually]. Ensure your application is configured to accept connections on port 8081.

[NOTE]

> To access a Runtime Fabric instance using more than one DNS, add additional DNS entries in the SAN certificate property. If a certificate has multiple DNS entries specified in the SAN property, the available URLs are displayed in the *Application url* section.

Reviewer: ```suggestion
To access a Runtime Fabric instance using more than one FQDN, add additional DNS entries in the SubjectAltName (SAN) certificate property. If a certificate has multiple DNS entries specified in the SAN property, the available URLs are displayed in the *Public endpoint* section. Each deployed application can only be addressed using a single FQDN.
```

. Select *Ingress*.

> . From the *Domain* drop-down list, choose the appropriate domain.

Reviewer: ```suggestion
. Enable the *Enable inbound traffic* checkbox.
. From the *Domain* drop-down list, choose the appropriate domain.
```

+

> . Enable or disable the following options:

Reviewer: ```suggestion
. Optionally, enable or disable the following options:
```

. To preview the endpoint, click the generated preview link.
. To add additional endpoints, click *+ Add Endpoint*.

=== Configure a Public Endpoint for the Application for VMs/Bare Metal

Use the following steps to configure a public endpoint and network configuration for an application deployed to Runtime Fabric for VMs/Bare Metal.

> You must first xref:enable-inbound-traffic.adoc[enable inbound traffic manually]. Ensure your application is configured to accept connections on port 8081.

Reviewer 2: You must first xref:enable-inbound-traffic.adoc[enable inbound traffic manually]. Ensure that your application is configured to accept connections on port 8081.
